src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Replacements/DateTimeOffsetConverter2.cs

```diff
@@ -55,7 +55,7 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
                 Type intType = typeof(int);
                 ConstructorInfo constructor = typeof(DateTimeOffset).GetConstructor(
-                    new Type[] {
+                    new[] {
```

Comment:
Do not make the `{` in the lastest character in the line

Reply:
@lindexi, it was the original formatting. Should I fix it as well in the current pull request?